webpack.config.js

```diff
@@ -0,0 +1,53 @@
+var path = require("path");
+
+module.exports = {
+  entry: {
+    app: [
+      './src/index.js'
+    ]
+  },
+
+  output: {
+    path: path.resolve(__dirname + '/dist'),
+    filename: '[name].js',
+    publicPath: '/'
+  },
+
+  module: {
+    loaders: [
+      {
+        test: /\.(css|scss)$/,
+        loaders: [
+          'style-loader',
+          'css-loader',
+        ]
+      },
+      {
+        test:    /\.html$/,
+        exclude: /node_modules/,
+        loader:  'file?name=[name].[ext]',
+      },
+      {
+        test:    /\.elm$/,
+        exclude: [/elm-stuff/, /node_modules/],
+        loader: 'elm-hot!elm-webpack',
```

Comment:
What do you think about being strictly consistent with how you declare the loaders? i.e consistently including/excluding the `-loader` suffix and consistently using an array or `!` for chaining loaders.